server-common/src/main/java/org/apache/kafka/server/group/share/PersisterStateManager.java

```diff
@@ -390,7 +468,7 @@ protected boolean isRequestResponse(ClientResponse response) {
     }
 
     @Override
-    protected void handleRequestResponse(ClientResponse response) {
+protected void handleRequestResponse(ClientResponse response) {
       log.info("Read state response received. - {}", response);
```

Comment:
Soon all of these `log.info` will have to be replaced by something less intrusive. For now, while the code is still being brought up for the first time, I am happy to let it remain like this.